### ISSUE TITLE
demonstrate that BLISS outperforms the LSST pipeline at cataloging DC2

### ISSUE THREAD

**Issue by jeff-regier** [prob-ml/bliss#898]
Some changes will likely be required to achieve this result:

- [x] train with randomly selected 80x80 DC2 subimages, not just one 800x800 subimage per batch, for better regularization via batchnorm
- [x] turn on multiband processing for DC2
- [x] turn on preprocessing of the images with multiple types of log transformations
- [x] hyperparameter tuning, especially for learning rate. We want to see a fairly smooth U-shaped validation loss curve
- [x] turn on training with variable PSFs, with both deconvolved images and PSF parameters appended to the encoder input
- [x] try the new rolling z-score image transformation with and without various log transformations
- [x] #904
